docs/standard/datetime/datetime-support-in-system-text-json.md

```diff
@@ -0,0 +1,318 @@
+| `time-second`     | `ss`                                                       | 4.3.10a              | 00-58, 00-59, 00-60 based on leap second rules                                                                  |
+| `time-secfrac`    | `s`                                                        | 5.3.14               | There must be one digit, but the max number of digits is implemenation defined                                  |
+| `time-numoffset`  | `("+" / "-") time-hour ":" time-minute`                    |                      |                                                                                                                 |
+| `time-offset`     | `"Z" / time-numoffset`                                     |                      |                                                                                                                 |
+| `partial-time`    | `time-hour ":" time-minute ":" time-second [time-secfrac]` |                      |                                                                                                                 |
+| `full-date`       | `date-fullyear "-" date-month "-" date-mday`               | 5.2.2.1b             | Extended calendar date                                                                                          |
+| `full-time`       | `partial-time time-offset`                                 | 5.3.3 or 5.3.4.2     | 5.3.3 is "UTC of day" while 5.3.4.2 is "Local time of day with the time shift between local time scale and UTC" |
+| `date-time`       | `full-date "T" full-time`                                  | 5.4.2.1              | Extended calendar date and time of day                                                                          |
+
+## Extended ISO 8601-1:2019 Profile in <xref:System.Text.Json>
+
+### Parsing
+
+The extended ISO 8601 profile implemented in System.Text.Json uses these components to define the following levels of date-time granularity:
+
+1. `full-date` (ISO 8601-1:2019 5.2.2.1)
+	1. `YYYY-MM-DD`
+
+ISO 8601-1:2019 5.2.2.2 "Representations with reduced precision" allows for just YYYY-MM (a) and just YYYY (b), but we currently don't permit it.
+
+2. `full-date "T" time-hour ":" time-minute`
+	1. `YYYY-MM-DDThh:mm`
+
+3. `full-date "T" partial-time`
+	1. `YYYY-MM-DDThh:mm:ss`
+	2. `YYYY-MM-DDThh:mm:ss.s`
+
+4. `full-date "T" time-hour ":" time-minute time-offset`
+	1. `YYYY-MM-DDThh:mmZ`
+	2. `YYYY-MM-DDThh:mm("+" / "-")hh":"mm`
+
+5. `date-time` (ISO 8601-1:2019 5.4.2.1)
+	1. `YYYY-MM-DDThh:mm:ssZ`
+	2. `YYYY-MM-DDThh:mm:ss.sZ`
+	3. `YYYY-MM-DDThh:mm:ss("+" / "-")hh":"mm`
+	4. `YYYY-MM-DDThh:mm:ss.s("+" / "-")hh":"mm`
+
+"T" is optional per spec, but _only_ when times are used alone. In our case, we're reading out a complete date & time and as such require "T" (ISO 8601-1:2019 5.4.2.1b).
```

Comment:
I don't think that this sentence really adds value (again, it is useful as a code comment, but not as a doc).

```suggestion
```